cmd/editorconfig-checker/main.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/editorconfig-checker/editorconfig-checker/v3/pkg/error"
 	"github.com/editorconfig-checker/editorconfig-checker/v3/pkg/files"
 	"github.com/editorconfig-checker/editorconfig-checker/v3/pkg/logger"
+	"github.com/editorconfig-checker/editorconfig-checker/v3/pkg/output_formats"
```

Comment:
snake_case in a package name :grimacing: 

https://go.dev/blog/package-names

> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. They are often simple nouns, such as:

Reply:
Okay, would `outputformats` be a suitable name?

Reply:
`format` would be simpler and more classic

but `outputformat` is OK (but singular)

Yes, there is already package with a plural, but they are not respecting the idiomatic way to name them

> Package names are usually singular except the cases when they collide with reserved words like strings, types, etc.

Reply:
fixed